tests/testConvolve.py

```diff
@@ -39,7 +39,7 @@
 import numpy
 
 import lsst.utils
-import lsst.utils.tests as utilsTests
+import lsst.utils.tests
 import lsst.pex.logging as pexLog
 import lsst.afw.geom as afwGeom
 import lsst.afw.image as afwImage
```

Comment:
I've been wondering about this too. I know that Jenkins caches a ticket branch for future tests, so is it possible that someone ran a test before the filename was changed and that Jenkins didn't bother to remove the old kernel.pyc file either? I'm really not sure but this seems like a question we might want to have answered for the future.